app/package.json

```diff
@@ -81,6 +81,7 @@
     "@types/react-addons-test-utils": "^0.14.18",
     "@types/react-dom": "^15.5.0",
     "@types/react-redux": "^5.0.14",
+    "@types/react-router": "4.0.25",
```

Comment:
The latest version of these typings (which are a dependency of the react typings) seem to have a bug. See failing TC build on master. So have pinned this version here.